exercises/ledger/canonical-data.json

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
Not sure if this file is intended to be part of this change?

GitHub shows the whole file as new, but the description of the change means it seems like only the one extra line in the lycheeigbore should be expected?

Reply:
Thanks! I somehow got that other file/commit on my local/main. It was not meant to be included. It's been dropped.